### PR TITLE
spawn proper worker for metadata proxy

### PR DESCRIPTION
**Pull request by markmcclain**

### CODE REVIEW

**Comment by rodis** [comment]
LGTM

File: akanda/rug/main.py [hunk 1]
```diff
@@ -145,6 +148,13 @@ def main(argv=sys.argv[1:]):
     )
     notification_proc.start()
 
+    metadata_proc = multiprocessing.Process(
+        target=metadata.serve,
+        args=(quantum_api.get_local_service_ip(cfg.CONF).split('/')[0],),
```
Comment (dhellmann) [inline akanda/rug/main.py]: What is this expression doing?

**Comment by dhellmann** [comment]
I opened DHC-1561 to move the metadata proxy service out of the rug process. There's no reason it can't be its own process, right?

File: akanda/rug/main.py (hunk 1 above)
Comment (markmcclain) [inline akanda/rug/main.py]: Getting the IP address.  Obvious isn't it? :)

**Comment by markmcclain** [comment]
This needs to be a child process of the rug because there is a timing issue to ensure that the management port is properly created on the rug host.

File: akanda/rug/main.py (hunk 1 above)
Comment (dhellmann) [inline akanda/rug/main.py]: Call it a rhetorical question.

**Pull request merged**